.pre-commit-config.yaml

```diff
@@ -27,21 +27,13 @@ repos:
     - id: check-executables-have-shebangs
     - id: check-yaml
 
--   repo: https://github.com/psf/black-pre-commit-mirror
-    rev: 24.3.0
-    hooks:
-    - id: black-jupyter
-
--   repo: https://github.com/adamchainz/blacken-docs
```

Comment:
Sorry if I missed this information - but is ruff also running on docs? I couldn't find the confirmation online. The closest I got was https://github.com/astral-sh/ruff/issues/8237, but then I don't see where ruff is configured to run on the doc?

Reply:
hum. I may have tricked myself into believing that it was supported when I misunderstood something and didn't link the source in https://github.com/yt-project/yt/issues/4748#issuecomment-1971112987

I *think* the source of my confusion was https://github.com/astropy/astropy/pull/16132, which doesn't have anything to do with embedded python blocks.
I'll revert this and keep blacken-docs around for a little longer, but I'll also link the ruff issue. Thanks !